packages/wasi/src/index.ts

```diff
@@ -636,17 +636,16 @@ export default class WASIDefault {
         this.view.setBigUint64(bufPtr, BigInt(rstats.ino), true);
         bufPtr += 8;
         this.view.setUint8(bufPtr, stats.filetype);
-        bufPtr += 4;
-        this.view.setUint32(bufPtr, Number(rstats.nlink), true);
-        bufPtr += 4;
+        bufPtr += 8;
```

Comment:
Why this is changed?

Reply:
Since `linkcount` is (now) 64 bits, it should be 8-byte aligned rather than 4-byte aligned.